x-pack/plugins/security_solution/server/lib/telemetry/filterlists/prebuilt_rules_alerts.ts

```diff
@@ -9,7 +9,54 @@ import type { AllowlistFields } from './types';
 
 export const prebuiltRuleAllowlistFields: AllowlistFields = {
   _id: true,
+  id: true,
   '@timestamp': true,
+  // Base alert fields
+  'kibana.alert.ancestors': true,
+  'kibana.alert.depth': true,
+  'kibana.alert.original_event.action': true,
+  'kibana.alert.original_event.category': true,
+  'kibana.alert.original_event.dataset': true,
+  'kibana.alert.original_event.kind': true,
+  'kibana.alert.original_event.module': true,
+  'kibana.alert.original_event.type': true,
+  'kibana.alert.original_time': true,
+  'kibana.alert.reason': true,
+  'kibana.alert.risk_score': true,
+  'kibana.alert.rule.actions': true,
+  'kibana.alert.rule.category': true,
+  'kibana.alert.rule.consumer': true,
+  'kibana.alert.rule.created_at': true,
+  'kibana.alert.rule.description': true,
+  'kibana.alert.rule.enabled': true,
+  'kibana.alert.rule.exceptions_list': true,
+  'kibana.alert.rule.execution.uuid': true,
+  'kibana.alert.rule.false_positives': true,
+  'kibana.alert.rule.from': true,
+  'kibana.alert.rule.immutable': true,
+  'kibana.alert.rule.interval': true,
+  'kibana.alert.rule.name': true,
+  'kibana.alert.rule.producer': true,
+  'kibana.alert.rule.references': true,
+  'kibana.alert.rule.risk_score_mapping': true,
+  'kibana.alert.rule.rule_id': true,
+  'kibana.alert.rule.rule_type_id': true,
+  'kibana.alert.rule.severity': true,
+  'kibana.alert.rule.severity_mapping': true,
+  'kibana.alert.rule.tags': true,
+  'kibana.alert.rule.threat': true,
+  'kibana.alert.rule.timestamp_override': true,
+  'kibana.alert.rule.type': true,
+  'kibana.alert.rule.updated_at': true,
+  'kibana.alert.rule.uuid': true,
+  'kibana.alert.rule.version': true,
+  'kibana.alert.severity': true,
+  'kibana.alert.status': true,
+  'kibana.alert.uuid': true,
+  'kibana.alert.workflow_status': true,
+  'kibana.space_ids': true,
+  'kibana.version': true,
```

Comment:
@terrancedejesus it'll get all of the "context" if it's in these fields. The "filterlists" concept just describes an allowlist/whitelist we use to approve fields to be sent up. The gist doc I posted above is _all the fields, but we will drop many of those if they're not in this file. Does that make sense?